infrastructure/validators/onboarding/run-validator/polkadot.md

```diff
@@ -0,0 +1,873 @@
+
+#### Installation from official releases
+
+The official binaries can be downloaded from the
+[Github Releases](https://github.com/paritytech/polkadot-sdk/releases). You should download the
+latest available version. You can also download the binaries by using the following direct links
+(replace X.Y.Z by the appropriate version):
+
+```sh
+https://github.com/paritytech/polkadot-sdk/releases/download/polkadot-vX.Y.Z/polkadot
+https://github.com/paritytech/polkadot-sdk/releases/download/polkadot-vX.Y.Z/polkadot-execute-worker
+https://github.com/paritytech/polkadot-sdk/releases/download/polkadot-vX.Y.Z/polkadot-prepare-worker
+```
+
+#### Optional: Installation with Package Managers
+
+The Polkadot Binary in included in `Debian` derivatives (i.e. **Debian**, **Ubuntu**) and
+`RPM-based` distros (i.e. **Fedora**, **CentOS**).
+
+#### Debian-based (Debian, Ubuntu)
+
+Run the following commands as the root user:
+
+```ssh
+# Import the security@parity.io GPG key
+gpg --recv-keys --keyserver hkps://keys.mailvelope.com 9D4B2B6EB8F97156D19669A9FF0812D491B96798
+gpg --export 9D4B2B6EB8F97156D19669A9FF0812D491B96798 > /usr/share/keyrings/parity.gpg
+# Add the Parity repository and update the package index
+echo 'deb [signed-by=/usr/share/keyrings/parity.gpg] https://releases.parity.io/deb release main' > /etc/apt/sources.list.d/parity.list
+apt update
+# Install the `parity-keyring` package - This will ensure the GPG key
+# used by APT remains up-to-date
+apt install parity-keyring
+# Install polkadot
+apt install polkadot
+```
+
+#### RPM-based (Fedora, CentOS)
+
+Run the following commands as the root user:
+
+```bash
+# Install dnf-plugins-core (This might already be installed)
+dnf install dnf-plugins-core
+# Add the repository and enable it
+dnf config-manager --add-repo https://releases.parity.io/rpm/polkadot.repo
+dnf config-manager --set-enabled polkadot
+# Install polkadot (You may have to confirm the import of the GPG key, which
+# should have the following fingerprint: 9D4B2B6EB8F97156D19669A9FF0812D491B96798)
+dnf install polkadot
+```
+
+Make sure you verify the installation (see the "Verify the installation" section).
+
+!!!note "By default, the Polkadot systemd service is disabled"
+    To start the service, run:
+    ```bash
+    sudo systemctl start polkadot.service
+    ```
+
+#### Optional: Installation with Ansible
+
+To manage Polkadot installation with Ansible, you can use the **Substrate node role** distributed on
+the [Parity chain operations Ansible collection](https://github.com/paritytech/ansible-galaxy/)
+
+#### Optional: Installation with Docker
+
+To run Polkadot on a Docker or an OCI compatible container runtime, you can use the official
+[parity/polkadot docker image](https://hub.docker.com/r/parity/polkadot/tags), available on Docker
+Hub (replace X.Y.Z by the appropriate version):
+
+```sh
+docker.io/parity/polkadot:vX.Y.Z
+```
+
+### Optional: Building the Polkadot binaries from sources
```

Comment:
This section should be either removed or it can be replaced by a link to the source build instructions at https://github.com/paritytech/polkadot-sdk/tree/master/polkadot#building

Reply:
Done, thank you